language/move-stdlib/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "move-stdlib"
 version = "0.1.1"
-edition = "2018"
+edition = "2021"
 authors = ["Diem Association <opensource@diem.com>"]
 description = "Diem stdlib"
 repository = "https://github.com/diem/diem"
```

Comment:
Unrelated but we should probably change this lol

Reply:
Looks like all of our crates still have these... Let's do a codemod and get them replaced